scripts/webregautoin/README.md

```diff
@@ -12,20 +12,14 @@ credentials and then automatically select the `Remember me for 7 days` checkbox
 That way, you don't need to worry about having to authenticate via Duo for the next 7 days.
 
 ## Authentication Modes
-As of recently, this script supports either Push or SMS mode. The modes only really matter at the beginning (i.e., when
-you start the script).
-- Push mode essentially means that, when the script is starting up, the script will initially authenticate you using 
-  Duo Push.
-- SMS mode means that, when the script is starting up, the script will initially use the SMS code that best fits the 
-  hint that is given (i.e., it will try to find the code you defined in the configuration file that satisfies the hint
-  "Your next SMS Passcode starts with XXX"). 
+Prior to March 26, 2024, this script supported either Push or SMS mode. Now, because of the new 
+[Duo Universal Prompt](https://blink.ucsd.edu/technology/security/services/two-step-login/universal-prompt.html),
+only Push is supported. The authentication mode only matters at the beginning.
```

Comment:
explain what "only matters at the beginning" means lol it sounds so cryptic 

maybe just switch to "the mode needs to specified so that on start up, the script will auth you using duo push"